### PR TITLE
Fixes #21912 - cs cv env ids for hostgroup

### PR DESCRIPTION
**Pull request by parthaa**
Make the hammer host group and host calls show id information for
content source, content view, lifecycle environment and kickstart
repository ids

### CODE REVIEW

**Comment by parthaa** [comment]
Before this PR
```
$ hammer hostgroup info --id=8 
Id:                     8
Name:   .....
....
Content View:           rhel7
Lifecycle Environment:  Library
Content Source:         capules
Repository:             Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.3
```

After this PR
```
$ hammer hostgroup info --id=8 
Id:                     8
Name:   .....
....
Content View:           
    ID:   27
    Name: rhel7
Lifecycle Environment:  
    ID:   1
    Name: Library
Content Source:         
    ID:   1
    Name: capules
Repository:             
    ID:   97
    Name: Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.3
```

**Comment by parthaa** [comment]
Before this PR
```
$ hammer host info --id=9 
Id:                     9
Name:   .....
....
Content Information:      
    Content View:           rhel7
    Lifecycle Environment:  Library
    Content Source:         capules
    Repository:             Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.3
   ........
```

After this PR
```
$ hammer host info --id=9 
Id:                     9
Name:   .....
....
Content Information:      
    Content View:          
        ID:   27
        Name: rhel7
    Lifecycle Environment: 
        ID:   1
        Name: Library
    Content Source:        
        ID: 1   
        Name: capules
    Repository:            
        ID:   97
        Name: Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.3
```

File: lib/hammer_cli_katello/hostgroup_extensions.rb [hunk 1]
```diff
@@ -56,10 +56,25 @@ module HostgroupExtensions
...
+          field :lifecycle_environment_id, _("ID")
+          field :lifecycle_environment_name, _("Name")
+        end
+
+        label _("Content Source") do
+          field :content_source_id, _("ID")
+          field :content_source_name, _("Name")
+        end
+
+        label _("Repository") do
```
Comment (akofink) [inline lib/hammer_cli_katello/hostgroup_extensions.rb]: Label this 'Kickstart Repository'

File: lib/hammer_cli_katello/host_extensions.rb [hunk 2]
```diff
@@ -33,10 +33,26 @@ module HostExtensions
...
+              field :lifecycle_environment_id, _("ID")
+              field :lifecycle_environment_name, _("Name")
+            end
+
+            label _("Content Source") do
+              field :content_source_id, _("ID")
+              field :content_source_name, _("Name")
+            end
+
+            label _("Repository") do
```
Comment (akofink) [inline lib/hammer_cli_katello/host_extensions.rb]: 'Kickstart Repository'

**Pull request merged**